tools/importer-rest-api-specs/components/terraform/schema/build_chaos_experiment_test.go

```diff
@@ -6,18 +6,19 @@ package schema
 import (
 	"testing"
 
+	"github.com/hashicorp/pandora/tools/data-api-sdk/v1/models"
```

Comment:
import ordering

Reply:
FWIW file this gets removed in #3913 since these tests are still disabled, so I'm going to leave this for the moment.